integration_test/third_party_apps_data/applications/tomcat/metadata.yaml

```diff
@@ -92,30 +92,58 @@ expected_logs:
     value_regex: org.apache.catalina.startup.Catalina.start
     type: string
     description: source of where the log originated
+  - name: jsonPayload.level
+    type: string
+    description: Log entry level
+    optional: true
+  - name: severity
+    type: string
+    description: ''
 - log_name: tomcat_access
   fields:
   - name: httpRequest.remoteIp
     value_regex: 127.0.0.1
     type: string
     description: ''
+    optional: true
   - name: httpRequest.requestUrl
     value_regex: /forbidden.html
     type: string
     description: ''
+    optional: true
   - name: httpRequest.protocol
     value_regex: HTTP/1.1
     type: string
     description: ''
+    optional: true
   - name: httpRequest.requestMethod
     value_regex: GET
     type: string
     description: ''
+    optional: true
+  - name: httpRequest.responseSize
+    type: string
+    description: ''
+    optional: true
+  - name: httpRequest.userAgent
+    type: string
+    description: ''
+    optional: true
+  - name: httpRequest.status
+    type: string
+    description: ''
+    optional: true
   - name: jsonPayload.host
     type: string
     description: Contents of the Host header
+    optional: true
   - name: jsonPayload.user
     type: string
-    description: Authenticated username for the request
```

Comment:
Is this intentionally being drop? @ridwanmsharif 

Based on this https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/pull/275/files was generated @EvanSimpson might be curious on why it's being dropped.

Reply:
I'll echo this question as well - is this intentional?